### PR TITLE
Add text on how to run it in a dir other than home

### PR DESCRIPTION
**Pull request by TheOtherRealm**
## References
https://stackoverflow.com/questions/46755604/how-to-navigate-to-a-different-directory-in-jupyter-notebook/46762386

### CODE REVIEW

File: docs/source/getting_started/starting.rst [hunk 1]
```diff
@@ -11,6 +11,15 @@ Start JupyterLab using:
...
 JupyterLab will open automatically in your browser.
 
+If the location of your notebook files are not in your home directory (more often the case in Windows, or if you have an external drive) you can use the `--notebook-dir` added parameter to start JupyterLab in that directory.  Be very careful though, never run it in your root volume (ex: `c/` in Windows, `/` in Linux) unless you know what you are doing and are prepared for the consequences! 
+
+.. code:: bash
+
+    #Windows Example
+    jupyter lab --notebook-dir E:/
+    #Linux Example
+    jupyter lab --notebook-dir /var/www/html/
```
Comment (fcollonval) [inline docs/source/getting_started/starting.rst:21]: Actually there is a shortcut for that option:

```suggestion
    jupyter lab <path/to/working/directory>
```

File: docs/source/getting_started/starting.rst [hunk 2]
```diff
@@ -11,6 +11,15 @@ Start JupyterLab using:
 
 JupyterLab will open automatically in your browser.
 
+If the location of your notebook files are not in your home directory (more often the case in Windows, or if you have an external drive) you can use the `--notebook-dir` added parameter to start JupyterLab in that directory.  Be very careful though, never run it in your root volume (ex: `c/` in Windows, `/` in Linux) unless you know what you are doing and are prepared for the consequences!
```
Comment (fcollonval) [inline docs/source/getting_started/starting.rst:14]: The default command opens the current folder. So I would rephrase the sentence (+ taken into account the next comment).

```suggestion
If your notebook files are not in the current directory, you can pass your working directory path as argument when starting JupyterLab. Avoid running it from your root volume (e.g. `C:\` on Windows or `/` on Linux) to limit the risk of modifying system files.
```

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks for the suggestion @TheOtherRealm 

I left some comments.

File: docs/source/getting_started/starting.rst (hunk 1 above)
Comment (krassowski) [reply docs/source/getting_started/starting.rst:21]: Also, `--notebook-dir` is deprecated, but `--root-dir` and `--preferred-dir` might be worth mentioning. Combining the two allows to set root to e.g. `/home/user/projects/` but open the browser or `/home/user/projects/current_project/`.

Comment (TheOtherRealm) [reply docs/source/getting_started/starting.rst:21]: `--root-dir' and `--preferred-dir` do not seem to work but `--app_dir` and `--preferred_dir` do. I'm running v3.2.1 .

Comment (TheOtherRealm) [reply docs/source/getting_started/starting.rst]: I updated the pending changes to reflect the new parameters

**Review by fcollonval — APPROVED** [review]
Thanks @TheOtherRealm

**Comment by jtpio** [comment]
Looks like this PR is targeting the `3.2.x` branch?

Maybe we can update it to target `master`, and then backport to `3.2.x`.

**Comment by jtpio** [comment]
> Maybe we can update it to target master, and then backport to 3.2.x.

Let's just backport back to `master` when merged.

Thanks!

**Comment by jtpio** [comment]
@meeseeksdev please backport to master
